### PR TITLE
minor polish and text edits

### PR DESCRIPTION
**Pull request by lvuch**
advanced settings to settings:
https://github.com/rancher/dashboard/issues/3527

### CODE REVIEW

File: assets/translations/en-us.yaml [hunk 1]
```diff
@@ -5775,7 +5775,7 @@ typeLabel:
   management.cattle.io.setting: |-
     {count, plural,
       one { Advanced Setting }
```
Comment (nwmac) [inline assets/translations/en-us.yaml:5777]: Shouldn't this also be changed to Settings ?

**Review by nwmac — CHANGES_REQUESTED** [review]

File: assets/translations/en-us.yaml (hunk 1 above)
Comment (lvuch) [reply assets/translations/en-us.yaml:5777]: hm I didn't catch this one since it's singular? When does it appear?

**Comment by nwmac** [comment]
Ir probably doesn't - but if there was only one setting, it would.

**Review by nwmac — APPROVED** [review]
LGTM

**Review by nwmac — APPROVED** [review]

**Pull request merged**